Source/DataSources/ModelVisualizer.js

```diff
@@ -235,9 +269,7 @@ define([
     }
 
     function onModelReady(model) {
-        model.activeAnimations.addAll({
```

Comment:
Commit 9359a01 fixes this issue by providing a runAnimations property on ModelGraphics